### PR TITLE
Add compass-interface-core

### PR DESCRIPTION
**Pull request by AA-Turner**
Adding compass-interface-core from pypi

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AA-Turner** [comment]
@conda-forge-admin, please ping team

Ready for review!

File: recipes/compass-interface-core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (chrisburr) [inline recipes/compass-interface-core/meta.yaml:1]:
```suggestion
```

File: recipes/compass-interface-core/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+about:
+  home: https://github.com/the-scouts/compass-interface-core
+  license: MIT
+  license_file: LICENSE
+  summary: The unofficial API to the TSA Compass membership database
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```
Comment (chrisburr) [inline recipes/compass-interface-core/meta.yaml:41]:
```suggestion
```

File: recipes/compass-interface-core/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - AA-Turner
+    - rglss
```
Comment (chrisburr) [inline recipes/compass-interface-core/meta.yaml:43]: @rglss Are you happy to be listed as a feedstock maintainer here?

Comment (rglss) [reply recipes/compass-interface-core/meta.yaml:41]: @chrisburr Yup, thanks for checking!

Comment (AA-Turner) [reply recipes/compass-interface-core/meta.yaml:41]: @chrisburr also noting for avoidance of doubt I'm happy to be a maintainer here also

**Comment by AA-Turner** [comment]
Thanks @chrisburr :)

**Pull request merged**